src/ProjectTemplates/Web.Spa.ProjectTemplates/Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj

```diff
@@ -6,6 +6,8 @@
     <Description>Single Page Application templates for ASP.NET Core</Description>
     <PackageTags>$(PackageTags);spa</PackageTags>
     <IsShippingPackage>true</IsShippingPackage>
+    <!-- By default Pack will exclude files that start with '.', we want to include those files -->
+    <NoDefaultExcludes>true</NoDefaultExcludes>
```

Comment:
@ryanbrandenburg @dougbu [NuGet fixed a bug](https://github.com/NuGet/NuGet.Client/pull/3114/) that was allowing files starting with "." to be packed. Since it looks like we want those files included we need to set this value.